guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -47,23 +52,37 @@ If you customize configuration files, manually or use a tool such as Hiera, thes
 [[upgrade_paths]]
 == Upgrade Paths
 
-You can upgrade to {ProjectName} from the previous version. {ProjectServer}s and {SmartProxyServer}s on earlier versions must first be upgraded to the {Project} version previous to this. For more information, see the {Project} {ProductVersionPrevious} Upgrading and Updating {ProjectName} guide.
+ifdef::satellite[]
+You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}. {ProjectServer}s and {SmartProxyServer}s on earlier versions must first be upgraded to {Project} {ProjectVersionPrevious}. For more details, see the {Project} {ProjectVersionPrevious} https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html/upgrading_and_updating_red_hat_satellite/[Upgrading and Updating {ProjectName}] guide.
+endif::[]
 
-.Overview of {Project} {ProductVersion} Upgrade Paths
-image::satellite_6.4_upgrade_paths.png[Overview of {Project} {ProductVersion} Upgrade Paths]
+ifndef::satellite[]
+You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}. {ProjectServer}s and {SmartProxyServer}s on earlier versions must first be upgraded to {Project} {ProjectVersionPrevious}. For more details, see the {Project} {ProjectVersionPrevious} Upgrade documentation.
+endif::[]
+
+.Overview of {Project} {ProjectVersion} Upgrade Paths
+ifdef::satellite[]
+image::satellite_6.4_upgrade_paths.png[Overview of {Project} {ProjectVersion} Upgrade Paths]
 
 WARNING: Upgrading from the Beta to GA version is not supported.
+endif::[]
 
-The high level steps in upgrading to {Project} {ProductVersion} are as follows.
+The high level steps in upgrading to {Project} {ProjectVersion} are as follows.
 
 . Clone your existing {ProjectServer}s. For more information, see xref:cloning_satellite_server[].
```

Comment:
```suggestion
ifdef::satellite[]
. Clone your existing {ProjectServer}s. For more information, see xref:cloning_satellite_server[].
endif::[]
```

This is broken as there are no cloning instructions included in the Katello upgrade guide?